go.mod

```diff
@@ -11,27 +11,29 @@ require (
 	github.com/aws/aws-sdk-go v1.16.20 // indirect
 	github.com/fsnotify/fsnotify v1.4.7
 	github.com/go-openapi/spec v0.19.3
-	github.com/golang/protobuf v1.3.2
+	github.com/golang/protobuf v1.5.2
```

Comment:
Not sure why we're upgrading protobuf. Seems like a big change for upgrading a logging library.